sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.0.1 (2002-02-17)
```

Comment:
You are going back in time to 2002 to release this???
App Config as a service won't be existing back then :(

Reply:
While we're changing dates, this will probably be the 18th given the holiday :wink:

Reply:
Oof. That was bad. Fixing and also bumping to 19th.